stattutor/public/js/stattutor.js

```diff
@@ -647,6 +647,37 @@ function gen_k_of_n_checkbox_feedback(exp, act, intro, studentTerms, itemName, i
     result += (wantedA.length>=nNeeded ? "" : (wantedA.length>0 ? " "+existOtherSf : (nNeeded>0 ? " "+exist1Sf : "")));	    return result;
 }
 /**
+ * Generate a feedback message for a choose-one-of-N question about selecting a statistical test.
+ * @param {string} act student's answer
+ * @param {Array<string>} tests all answers available; act must be one of these
+ * @param {Array<string>} studentTests student-friendly names for each statistical test
+ * @param {Array<string>} descTests descriptions of when to use each test
+ * @param {Array<string>} useTests short statement of the use of each test
+ * @param {string} intro prefix to all responses
+ * @param {string} finish suffix to all responses
+ * @return {string} feedback message
+ */
+function gen_1_of_n_stat_test_feedback(act, tests, studentTests, descTests, useTests, intro, finish) {
```

Comment:
I'm not seeing this function called w/in the JavaScript;
it looks like it's referenced in the BRD file though?

Reply:
Yes, each of these new JavaScript functions is called from within the .brd files to generate formulaic feedback messages.
```
$ cd stattutor/public/problem_files
$ grep -l gen_1_of_n_stat_test_feedback $(find m[0-9]_* -name "*.brd")
m7_auto_premiums/auto_premiums.brd
$ grep -l gen_1_of_n_feedback $(find m[0-9]_* -name "*.brd")
m4_cell_phones/cell_phones.brd
m4_cell_phones_1q/cell_phones_1q.brd
m5_extra_credit/extra_credit.brd
m5_extra_credit_2q/extra_credit_2q.brd
m6_low_birth_weight/low_birth_weight.brd
m7_auto_premiums/auto_premiums.brd
```